includes/emails/class-wc-payments-email-ipp-receipt.php

```diff
@@ -63,6 +60,9 @@ public function __construct() {
 			// Triggers for this email.
 			add_action( 'woocommerce_payments_email_ipp_receipt_notification', [ $this, 'trigger' ], 10, 3 );
 
+			// Call parent constructor.
```

Comment:
+1 from me. The `WC_Email` feels poorly designed and in no way the intended usage seems natural. So I would even include that it's the intended use of parent class rather than some hack. 

Context: Comments like this can be viewed as an explanation of a hack but here it's actually the intended use case.

Reply:
Done